codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -116,11 +120,41 @@
     HttpHeaders setTimeMillis(CharSequence name, long value);
 
     @Override
-    HttpHeaders set(TextHeaders headers);
+    HttpHeaders set(Headers<? extends CharSequence> headers);
 
     @Override
-    HttpHeaders setAll(TextHeaders headers);
+    HttpHeaders setAll(Headers<? extends CharSequence> headers);
 
     @Override
     HttpHeaders clear();
+
+    /**
+     * {@link #get(CharSequence)} and convert the result to a {@link String}.
+     * @param name the name of the header to retrieve
+     * @return the first header value if the header is found. {@code null} if there's no such header.
+     */
+    String getAndConvert(CharSequence name);
```

Comment:
@Scottmitch can we rename those `*Convert*` methods to something with `String`? Like `getAsString(name)`